### PR TITLE
Adds wizard potion bags: Git strikes back

### PR DESCRIPTION
**Pull request by Irradiation**
See #20072 for the info regarding the original PR.

tl;dr Inquisitor PR fucked head.dmi and I can't into git, so here's a working version maybe possibly.

:cl:
 * rscadd: Added potion bags to magi-vends. They can hold a high amount of potions & can be used as hats.

### CODE REVIEW

File: code/game/objects/items/weapons/storage/bags.dm [hunk 1]
```diff
@@ -470,3 +470,21 @@ obj/item/weapon/storage/bag/plasticbag/quick_store(var/obj/item/I)
...
+	desc = "Not too dissimilar to the fabled bag of alcohol. The wizard federation is not responsible for possible rainbow puking."
+	icon = 'icons/obj/pbag.dmi'
+	icon_state = "pbag"
+	item_state = "pbag"
+	body_parts_covered = FULL_HEAD|BEARD
+	slot_flags = SLOT_BELT | SLOT_HEAD
+	storage_slots = 50
+	fits_max_w_class = 3
+	max_combined_w_class = 200
+	w_class = W_CLASS_TINY
```
Comment (Kurfursten) [inline code/game/objects/items/weapons/storage/bags.dm]: This is the size class used by cigarettes, pills, and pens.

Comment (Irradiation) [reply code/game/objects/items/weapons/storage/bags.dm]: It's magic, I don't have to explain shit. But I can put it to small size at least.

**Comment by SonixApache** [comment]
>bag doesn't count as wizard hat

**Comment by Irradiation** [comment]
If you're going to try and be /fa/, pay the /no-robes/ price. Or just be a feg and pick blink like everyone and their mother while jaunt requires robes.

**Pull request merged**